part-spec/definitions.json

```diff
@@ -1,250 +1,339 @@
-                },
-                "voltageOptions": {
-                    "description": "list of voltage levels supported by a pin",
-                    "comment": "units of volts",
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/unit"
-                    }
-                },
-                "floatUnused": {
-                    "description": "description of whether pin can safely be floated if it is not used",
-                    "type": "boolean"
-                },
-                "externalComponents": {
-                    "description": "list of external component structures recommended to be attached to a pin",
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/externalComponents"
-                    }
-                }
-            }
+  "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/definitions.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "pinSpec": {
+      "type": "object",
+      "required": [
+        "terminalIdentifier",
+        "name"
+      ],
+      "properties": {
+        "terminalIdentifier": {
+          "description": "pin or ball number as defined by datasheet",
+          "type": "string"
+        },
+        "name": {
+          "description": "name given to the signal appearing at the terminal of a component",
+          "type": "string"
+        },
+        "standardized_name": {
+          "description": "standard name of pin",
+          "type": "string",
+          "enum": [
+            "drain",
+            "gate",
+            "source",
+            "anode",
+            "cathode",
+            "vdd",
+            "vss"
+          ]
+        },
+        "description": {
+          "description": "description of the signal appearing at the terminal of an electric/electronic component",
+          "type": "string"
+        },
+        "numberOfSupportedFunctions": {
+          "description": "the total number of functions supported by this pin",
+          "type": "number"
+        },
+        "functionProperities": {
+          "description": "list of function objects that can apply to an individual pin",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/pinFunction"
+          }
+        },
+        "vihMin": {
+          "description": "minimum voltage at which an input is read as high",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "vilMax": {
+          "description": "maximum voltage at which an input is read as low",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
+        "vol": {
+          "description": "voltage ouput low or minimum voltage of a high output",
+          "comment": "units of volts",
+          "$ref": "#/definitions/unit"
+        },
```

Comment:
We should probably add vihMax: The most positive (least negative) value of high-level input voltage for which operation of the logic element within specification limits is to be expected.
And vilMin: The least
positive (most negative) value of low-level input voltage for which operation of the logic element within specification limits is to be expected.